### DISCUSSION TITLE
It is mentioned that is compatiable  with Big Sur?

### DISCUSSION THREAD

**Discussion by titanhusker** [chris1111/Catalina-Recovery-HD-Partition#4] [Q&A]
I tried it and I could seem to duplicate it with success. I need to restore from a Time Machine backup, however, I am running into the same issues, many people are having with the mac reverting the whole system back to 10.2.6. In fact, when it does, I can't see either the backups or the internal drive. Kind of no point in having a complete backup if you can't restore it. I will be following, I appreciate your hard work, it rebuilt the recovery drive for Catalina.

**Comment by chris1111** [comment]
this is not for BS its work on BS to create a Catalina Recovery HD (Catalina SSD) or create a USB Catalina Recovery HD
See release note

**Comment by titanhusker** [comment]
I figured that, is a Big Sur edition even possible? I think most people would not have upgraded had they had known

On Feb 26, 2021, at 5:31 AM, chris1111 <notifications@github.com<mailto:notifications@github.com>> wrote:



this is not for BS its work on BS to create a Catalina Recovery HD (Catalina SSD) or create a USB Catalina Recovery HD
